operator/config/rbac/role.yaml

```diff
@@ -0,0 +1,68 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
should the file name be manager-role.yaml too?
so that it better indicates that this is about managing operator* resources?

Reply:
this is autogenerated by kubebuilder; I should investigate if the file name can be changed, but I would advice against this because then we should adapt all the chain up for kustomize.

BTW. The user want see this file name because all the yaml files under config/ should be packed into a unique file during the release

Reply:
ok, understood.